docs/CONTRIBUTING.md

```diff
@@ -10,13 +10,15 @@ Before creating an issue:
 * If they are, reach out and try to help. 
 
 ## Pull Requests
-Please submit pull requests in response to open issues. If you have a bug or feature, bring it up in issues first, to avoid two or more people working on the same thing.
+Please submit pull requests in response to open issues. If you have a bug or feature, bring it up in issues first, to avoid two or more people working on the same thing. Remember to rebase and run `npm run pretest` before submitting a pull request.
```

Comment:
I'm not sure if this is the right place to mention rebasing and runnig tests.
There's another section in this document called THE LIST OF THINGS THAT MUST BE DONE BY A CONTRIBUTOR BEFORE FILLING A PR`. Maybe that's the appropriate place. Also, it's better to run `test`, not `pretest`, since the `pretest` is included in the `test`.